app/services/api/external-api.ts

```diff
@@ -14,6 +14,27 @@ export function Singleton() {
 }
 
 /**
+<<<<<<< HEAD
```

Comment:
There's merge resolution markup in a couple places in this file.  Kind of hilariously it's only in the comments so it's not affecting compilation.

Reply:
Ouch, looks like my IDE didn't detect conflicts in the comments, lol